__tests__/secrets.test.js

```diff
@@ -0,0 +1,69 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const app = require('../lib/app');
+const request = require('supertest');
+const UserService = require('../lib/services/UserService');
+
+const admin = {
+  first_name: 'admin',
+  last_name: 'admin',
+  email: 'admin',
+  password: 'admin',
+};
+const sampleUser = {
+  first_name: 'Test',
+  last_name: 'Testerson',
+  email: 'test2@test.com',
+  password: '12345',
+};
+
+const registerAndLogin = async () => {
+  const agent = request.agent(app);
+  const user = await UserService.create(sampleUser);
+  await agent
+    .post('/api/v1/users/sessions')
+    .send({ email: sampleUser.email, password: sampleUser.password });
+  return [agent, user];
+};
+
+describe('secrets', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('GET /api/v1/secrets should return a list of secrets if authd ', async () => {
```

Comment:
it'd be good to also check that you canNOT get the secrets when not auth'd

edit: I see you did that as part of the users test but since it's an action related to GETting the secrets route it belongs here